### ISSUE TITLE
Camera intrinsics for projection matrix

### ISSUE THREAD

**Issue by YuxuanSnow** [PerceivingSystems/bedlam_render#4]
Hi!

thanks for releasing the scripts.

I have a question regarding the camera intrinsics. I found the groundtruth unreal camera has following intrinsics:
- focal length: 36.905
- sensor width: 36
- sensor height: 20.25
- hfov: 52

However, when i create the following intrinsics matrix, the unprojected depth point cloud looks like a plane:
```
K = [[36.905, 0, 36],
     [0, 36.905, 20.5],
     [0, 0, 1]]
```

Could you refer me a script on how to compute the intrinsics matrix correctly?

Thanks!

**Comment by pixelite1201** [comment]
Hi,
You could refer to following [script](https://github.com/pixelite1201/BEDLAM/blob/master/data_processing/df_full_body.py#L72) to generate camera intrinsic matrix.

**Comment by tpsmpi** [comment]
Sensor width/height is always fixed at 36.0mm/20.25mm (16:9 DSLR sensor). 
HFOV is not always 52, it can also be 65 or in the case of the `zoom` subsets vary during a sequence. The dataset `ground_truth` folder `.csv` files provide the relevant camera information including HFOV for each image.

**Comment by YuxuanSnow** [comment]
> 

Thanks for reply. As far as i can see in the script that Priyanka provided, the hfov doesn't affect the projection matrix, is that correct?
